validator-tests/errormessage-hidden-removed.html

```diff
@@ -0,0 +1,37 @@
+<html lang="en-US">
+<head><title>When aria-errormessage is pertinent, authors MUST ensure the content is not hidden so users can navigate to and examine the error message</title></head>
+<body>
+
+<!--
+URL: https://www.w3.org/TR/wai-aria-1.2/#aria-errormessage
+RULE: " Authors MUST use aria-invalid in conjunction with aria-errormessage and when aria-errormessage is pertinent, authors MUST ensure the content is not hidden so users can navigate to and examine the error message"
```

Comment:
Question, potential blocker: Should an example be included that has no `aria-invalid` attribute? 

From the spec, emphasis mine:

> Initially, the object is in a valid state and either has aria-invalid set to false **or no aria-invalid attribute**, and the element referenced by aria-errormessage is not applicable.

Reply:
I think we would if this were a test file for all of aria-errormessage. 

However this is only a testfile for the added authors MUST statements in aria 1.2. Without aria-invalid, aria-errormessage is not pertinent so the entire sentence would not apply.